packages/contracts-bedrock/scripts/deploy/Deploy.s.sol

```diff
@@ -678,7 +678,7 @@ contract Deploy is Deployer {
                 _save: this,
                 _salt: _implSalt(),
                 _name: "SuperchainConfig",
-                _args: abi.encodeCall(ISuperchainConfig.__constructor__, ())
+                _args: DeployUtils.encodeConstructor(abi.encodeCall(ISuperchainConfig.__constructor__, ()))
```

Comment:
This would be a good semgrep rule, should we add this to the tracker in https://github.com/ethereum-optimism/client-pod/issues/846?

Reply:
Yes, I can also make this a hard rule in ci for now